.github/workflows/ci.yml

```
@@ -74,99 +123,161 @@ jobs:
+                if command -v "$tool" >/dev/null 2>&1; then
+                  echo "✓ $tool ($(command -v $tool))"
+                else
+                  echo "✗ $tool MISSING" && exit 1
+                fi
+              done
+              echo "=== All core tools present ==="
+            '
+
   # ── Linux aggregator: required status check ─────────────────────────────────
   # Branch protection on `main` requires a status check named `linux`. This job
+<<<<<<< HEAD
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)